checks/microbenchmarks/mpi/osu/osu_tests.py

```diff
@@ -240,11 +235,11 @@ class osu_build_run(osu_benchmark):
     def set_valid_systems_envs(self):
         build_type = self.osu_binaries.build_type
         if build_type == 'cuda':
-            self.valid_systems = ['+remote +nvgpu -uenv']
+            self.valid_systems = ['+remote +nvgpu']
```

Comment:
Do we want to run this in all environemts that have mpi and cuda? I am thinking if we should remove it from uenvs, like the non cuda version.

Reply:
Ah ok, now I get what you said. You are right. We just need the mpi / mpi and cuda